packaging/rpm/temboard-agent.spec

```diff
@@ -22,19 +22,9 @@ Source2:       temboard-agent.service
 Source3:       temboard-agent.rpm.conf
 BuildArch:     noarch
 Requires:      openssl
-%if 0%{?rhel} <= 6
-Requires:      python-argparse
-Requires:      python-logutils
-%endif
-%if 0%{?rhel} < 8
-Requires:      python-setuptools
-Requires:      python-psycopg2
-BuildRequires: python-setuptools
-%else
 Requires:      python3-setuptools
 Requires:      python3-psycopg2
 BuildRequires: python3-setuptools
```

Comment:
This will not work for centos7 because python3 packages are name as `python36-<name>`, so we'll need to keep an `%if 0%{?rhel} == 7` somewhere, I think.